skytemple_files/patch/handler/move_growth.py

```diff
@@ -67,15 +64,15 @@
                5000,1,1,1,
                   0,5,5,1]
 
-class MoveGrowthPatchHandler(AbstractPatchHandler):
+class MoveGrowthPatchHandler(AbstractPatchHandler, DependantPatch):
```

Comment:
i think this is a mistake?

Reply:
What's the problem?

Reply:
Oh nevermind, I didn't see the `depends_on` that was already there in the review screen.